bin/do_Pk1D.py

```diff
@@ -195,8 +194,8 @@ def compute_mean_delta(ll,delta,iv,zqso):
             if (d.mean_SNR<=args.SNR_min or d.mean_reso>=args.reso_max) : continue
 
             # first pixel in forest
-            for first_pixel in range(len(d.ll)) :
-                 if (sp.power(10.,d.ll[first_pixel])>args.lambda_obs_min) : break
+            for first_pixel,first_pixel_ll in enumerate(d.ll):
+                if (sp.power(10.,first_pixel_ll)>args.lambda_obs_min) : break
```

Comment:
could you remove the parenthesis? They are useless.

Reply:
while doing this, does sp.power add anything vs 10**ll?

Reply:
no, `10**ll` is very good.